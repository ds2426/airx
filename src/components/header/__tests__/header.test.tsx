import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../index';
import { expect, test } from 'vitest';

function toJson(component: renderer.ReactTestRenderer) {
  const result = component.toJSON();
  expect(result).toBeDefined();
  return result as renderer.ReactTestRendererJSON;
}

test('Footer renders', () => {
  const component = renderer.create(<Header />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
