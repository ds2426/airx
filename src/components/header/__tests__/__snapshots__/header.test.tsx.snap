// Vitest Snapshot v1

exports[`Footer renders 1`] = `
<div
  className="MuiBox-root css-1p89roy"
>
  <div
    className="MuiBox-root css-1xhj18k"
  >
    <img
      alt="aiRx"
      height="50"
      src="/src/assets/android-chrome-192x192.png"
      width="50"
    />
    <div
      className="MuiBox-root css-1go7of1"
    >
      ai
      <strong>
        R
      </strong>
      <sub>
        x
      </sub>
    </div>
  </div>
</div>
`;
